src/components/accountInitialization/index.js

```diff
@@ -34,7 +42,12 @@ class AccountInitialization extends React.Component {
         <p>{t('It is recommended that you initialize your Lisk ID.')}</p>
         <p>{t('The easiest way to do this is to send LSK to yourself. It will cost you only the usual {{fee}} LSK transaction fee.', { fee: fromRawLsk(fees.send) })}</p>
         <p>
-          <a target='_blank' href='https://help.lisk.io/account-security/should-i-initialize-my-lisk-account' rel='noopener noreferrer'>
+          <a
+            target='_blank'
+            href='https://help.lisk.io/account-security/should-i-initialize-my-lisk-account'
+            onClick={Piwik.trackingEvent('AccountInit', 'link', 'initializeMyLiskAccount')}
```

Comment:
```suggestion
            onClick={() => Piwik.trackingEvent('AccountInit', 'link', 'initializeMyLiskAccount')}
```
Otherwise you track the event when rendering the component - each time coming to "send" page.